tests/x509/test_x509.py

```diff
@@ -4728,15 +4728,15 @@ def test_tbs_certificate_bytes(self, backend):
             cert.signature_hash_algorithm,
         )
 
-    def test_verify_directly_issued_by_dsa(self):
+    def test_verify_directly_issued_by_dsa(self, backend):
```

Comment:
?

Reply:
We need these so that the skip logic for DSA applies (it's on the parent class)